packages/react-reconciler/src/__tests__/ReactSuspenseWithNoopRenderer-test.js

```diff
@@ -501,30 +501,18 @@ describe('ReactSuspenseWithNoopRenderer', () => {
 
     await rejectText('Result', new Error('Failed to load: Result'));
 
-    gate(flags => {
-      if (flags.enableSuspenseLayoutEffectSemantics) {
-        expect(Scheduler).toFlushAndYield([
-          'Error! [Result]',
-
-          // React retries one more time
-          'Error! [Result]',
-        ]);
-        expect(ReactNoop.getChildren()).toEqual([]);
-      } else {
-        expect(Scheduler).toFlushAndYield([
-          'Error! [Result]',
+    expect(Scheduler).toFlushAndYield([
+      'Error! [Result]',
 
-          // React retries one more time
-          'Error! [Result]',
+      // React retries one more time
+      'Error! [Result]',
 
-          // Errored again on retry. Now handle it.
-          'Caught error: Failed to load: Result',
-        ]);
-        expect(ReactNoop.getChildren()).toEqual([
-          span('Caught error: Failed to load: Result'),
-        ]);
-      }
-    });
+      // Errored again on retry. Now handle it.
+      'Caught error: Failed to load: Result',
+    ]);
+    expect(ReactNoop.getChildren()).toEqual([
+      span('Caught error: Failed to load: Result'),
+    ]);
```

Comment:
I guess this actually fixed something, too, though not sure why the fork in behavior was originally added